tests/publiccloud/register_system.pm

```diff
@@ -32,12 +32,17 @@ sub run {
         select_host_console();    # select console on the host, not the PC instance
 
         # note: ssh_script_retry dies on failure
-        $args->{my_instance}->retry_ssh_command("sudo SUSEConnect -r " . get_required_var('SCC_REGCODE'), timeout => 420, retry => 3);
+        my $regcode = get_required_var('SCC_REGCODE');
```

Comment:
Is `SCC_REGCODE` always there?

Reply:
It can be missing on onDemand images but those are excluded on top of this file.

Reply:
yes

Reply:
But all BYOS images we test we have `SCC_REGCODE` in my opinion.

Reply:
That should be fine, LGTM :rocket: